extension/manifest.json

```diff
@@ -1,12 +1,20 @@
 {
+    "applications": {
```

Comment:
`applications` key in the manifest is not recognized by chrome. We need to decide either to have a separate manifest for Firefox or comment this part by default.